style.tex

```diff
@@ -20,7 +20,7 @@
 \usepackage{courier}
 \usepackage{xparse}
 \usepackage{longtable}
-\usepackage{menukeys}
+\usepackage[os=win]{menukeys}
```

Comment:
would it be possible to make this option depend on the current version of the pdf you're building?